Makefile

```diff
@@ -0,0 +1,22 @@
+SCHEMA_FILES := $(shell find . -type f -path './schema/*.json' -not -path './.github/*' -not -path './node_modules/*' | sort)
+
+.PHONY: all
+all: install-tools compile-schema validate-example
+
+.PHONY: compile-schema
+compile-schema:
+	@if ! npm ls ajv-cli; then npm install; fi
+	@for f in $(SCHEMA_FILES); do \
+	    npx --no ajv-cli compile --spec=draft2020 -s $$f \
+	        || exit 1; \
+	done
+
+.PHONY: validate-example
+validate-example:
+	@if ! npm ls ajv-cli; then npm install; fi
+	npx --no ajv-cli validate --spec=draft2020 --errors=text -s ./schema/opentelemetry_configuration.json -d ./kitchen-sink-example.yaml
```

Comment:
It seems useful to not only validate that that the schema files compile, but that one or more canonical example files successfully validate against the schema. The example files can evolve along side the schema.

Reply:
I suppose not necessary at this stage but best to test all of the examples so none can get out of sync with the schema.

Reply:
Yup when there are more examples we should extend the validation.